packages/application-shell/src/components/navbar/devonly-menu-loader.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { wrapDisplayName } from 'recompose';
+
+const noop = () => Component => Component;
+
+const devonlyMenuLoader =
+  process.env.NODE_ENV !== 'development'
+    ? noop
+    : (getLoaderFn, mapMenuToProps) => Component => {
+        class WrappedComponent extends React.Component {
+          static displayName = wrapDisplayName(
```

Comment:
Nitpick: maybe not use recompose here as it's almost deprecated and we could aim to get rid of it in the app-shell maybe?

Reply:
I didn’t know that. What is the alternative then? We usually use it for `compose`

Reply:
Sorry, that came across wrong. I didn't mean to say "let's drop it" or that we decided. It was more of a suggestion: could the app-shell work without and if so what would be have to remove or custom build. Keep it for now. Should be a separate thing anyway. Sorry for the confusion.

Reply:
No worries, I understand your point. I was just asking what should we use to `compose`. Should we just copy the implementation and have it as an internal function?

Reply:
`flowRight` from lodash I suppose (the renamed compose to flowRight).

1. I would have these HoCs as little utils, yes
2. For the mc-fe it's almost unrealistic and not worthwhile getting rid of recompose

Reply:
FYI: https://www.apollographql.com/docs/react/api/react-apollo.html#compose